pylon-core/src/main/kotlin/io/github/pylonmc/pylon/core/event/PylonBlockLoadEvent.kt

```diff
@@ -0,0 +1,23 @@
+package io.github.pylonmc.pylon.core.event
+
+import io.github.pylonmc.pylon.core.block.PylonBlock
+import io.github.pylonmc.pylon.core.block.PylonBlockSchema
+import org.bukkit.block.Block
+import org.bukkit.event.Event
+import org.bukkit.event.HandlerList
+
+class PylonBlockLoadEvent(
+    val block: Block,
+    val pylonBlock: PylonBlock<PylonBlockSchema>,
+) : Event() {
+    override fun getHandlers(): HandlerList
+        = handlerList
+
+    companion object {
+        private val handlerList: HandlerList = HandlerList()
+
+        @JvmStatic
+        fun getHandlerList(): HandlerList
+                = handlerList
```

Comment:
```suggestion
	@JvmStatic
        val handlerList: HandlerList = HandlerList()
```
5 more secrets Java doesn't want you to know!